docs/index-server.md

```diff
@@ -119,6 +119,20 @@ be written to file.
 The user can set the location for these file by using 'carbon.indexserver.temp.path'. By default
```

Comment:
"location for these file" to be changed to "location for these files"

Reply:
done